_maps/shuttles/cargo_box.dmm

```diff
@@ -47,10 +48,8 @@
 	name = "Supply Shuttle Airlock";
 	req_access_txt = "31"
 	},
-/obj/docking_port/mobile/supply{
-	dwidth = 5;
-	width = 12
```

Comment:
Why did you remove these.

Reply:
StrongDMM removed them when saving back to TGM. Checking the  map post-save in alternate editors (FastDMM & Byond's built in editor) reveals that the values remain the same despite removal. Checkout the branch and check yourself. It compiles without issue and works ingame.

I have no idea 🤷‍♂ Default values probably?

Reply:
StrongDMM does remove default values IIRC.